internal/buildops/build_params_test.go

```diff
@@ -20,11 +20,12 @@ var _ = Describe("BuildParams", func() {
 
 	var _ = Describe("GetBuildResultsPath", func() {
 		var _ = DescribeTable("valid cases", func(module *mta.Module, expected string) {
-			path, _, err := GetBuildResultsPath(&dir.Loc{SourcePath: getTestPath("mtahtml5")}, module, expected)
+			loc := &dir.Loc{SourcePath: getTestPath("mtahtml5")}
+			path, _, _, err := GetModuleSourceArtifactPath(loc, false, module, expected)
 			Ω(err).Should(Succeed())
 			Ω(path).Should(HaveSuffix(expected))
```

Comment:
This check is weird when expected is an empty string like in the first test case

Reply:
removed